app/controllers/kaui/admin_allowed_users_controller.rb

```diff
@@ -103,13 +108,24 @@ def add_tenant
 
   private
 
+  # this will check if the user is managed externally or internally by a shiro config file.
```

Comment:
To make it clearer: `this will check if the user is managed by Kill Bill (in which case, the password is editable)`

Reply:
Yes it is. I pushed some changes to make it clearer.